kie-dmn-feel/src/main/java/org/kie/dmn/feel/lang/EvaluationContext.java

```diff
@@ -39,4 +41,8 @@
     Map<String, Object> getAllValues();
 
     FEELEventListenersManager getEventsManager();
+    
+    default void notifyEvt(Supplier<FEELEvent> event) {
```

Comment:
Limited default method, anyway represent a common operation which the user of the API/interface would make anyway.
Also to be noticed the `FEELEventListenersManager` is a type know from the method just above `getEventsManager()`